### DISCUSSION TITLE
using-source-generators-with-a-custom-attribute--to-generate-a-nav-component-in-a-blazor-app/

### DISCUSSION THREAD

**Discussion by giscus** [andrewlock/blog-comments#102] [Announcements]
# using-source-generators-with-a-custom-attribute--to-generate-a-nav-component-in-a-blazor-app/

In this post I extend my previous post and create a source generator that can specify the icon and order of items in a nav menu component

https://andrewlock.net/using-source-generators-with-a-custom-attribute--to-generate-a-nav-component-in-a-blazor-app/

**Comment by NateVolt** [comment]
There is a small typo in the first code block of the [Creating the source generator](https://andrewlock.net/using-source-generators-with-a-custom-attribute--to-generate-a-nav-component-in-a-blazor-app/#creating-the-source-generator) section. 

`<RestoreAdditionalProjectSources>https://dotnet.myget.org/F/roslyn/api/v3/index.json;$(RestoreAdditionalProjectSources)</storeAdditionalProjectSources>`
The closing tag is incorrect. It should be as follows.
`<RestoreAdditionalProjectSources>https://dotnet.myget.org/F/roslyn/api/v3/index.json;$(RestoreAdditionalProjectSources)</RestoreAdditionalProjectSources>`

I'm still reading the article, but I'm very grateful for the resources you provide for free to us all!

[a comment by an automated account is not shown]
